samples/oauth_rails_vetverification/README.md

```diff
@@ -0,0 +1,26 @@
+# README
+
+This is a sample app that will connect with the oauth provider to access Veteran Verification APIs.
+
+## Running locally
+
+Set up your API credentials at the [Developer Portal](https://developer.va.gov/apply).  The callback for this app to work needs to be `http://localhost:3000/callback`.
```

Comment:
Like that the callback is mentioned explicitly. This was the most confusing part for me when I was signing up!

Reply:
Nice work @erluti !!!